dk.tangramgames.mrrescue.json

```diff
@@ -0,0 +1,93 @@
+{
+  "app-id": "dk.tangramgames.mrrescue",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "mrrescue",
+  "finish-args": [
+    "--device=all",
+    "--socket=pulseaudio",
+    "--socket=x11"
+  ],
+  "modules": [
+    {
+      "name": "PhysicsFS",
+      "buildsystem": "cmake",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://icculus.org/physfs/downloads/physfs-3.0.1.tar.bz2",
+          "sha256": "b77b9f853168d9636a44f75fca372b363106f52d789d18a2f776397bf117f2f1"
+        }
+      ]
+    },
+    {
+      "name": "libmodplug",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/modplug-xmms/files/libmodplug/0.8.9.0/libmodplug-0.8.9.0.tar.gz",
+          "md5": "5ba16981e6515975e9a68a58d5ba69d1"
+        }
+      ]
+    },
+    {
+      "name": "luajit",
+      "buildsystem": "simple",
+      "build-commands": [
+        "make",
+        "make install PREFIX=/app"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://luajit.org/download/LuaJIT-2.0.5.tar.gz",
+          "sha256": "874b1f8297c697821f561f9b73b57ffd419ed8f4278c82e05b48806d30c1e979"
+        }
+      ]
+    },
+    {
+      "name": "love",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://bitbucket.org/rude/love/downloads/love-11.1-linux-src.tar.gz",
+          "sha256": "bc5e9a70021de830f1032d6afe900f0b81902e2b6c610e26cf1d6859bf8e7cde"
+        }
+      ]
+    },
+    {
+      "name": "mrrescue",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mv mrrescue /app/bin/mrrescue",
+        "install -Dm644 dk.tangramgames.mrrescue.desktop /app/share/applications/dk.tangramgames.mrrescue.desktop",
+        "install -Dm644 dk.tangramgames.mrrescue.appdata.xml /app/share/metainfo/dk.tangramgames.mrrescue.appdata.xml",
+        "install -Dm644 data/splash.png /app/share/icons/hicolor/256x256/apps/dk.tangramgames.mrrescue.png",
+        "zip -r /app/bin/mrrescue.love *"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/SimonLarsen/mrrescue.git",
+          "commit": "a5be73c60acb8d1be506f7b5e48e784492ba96ce"
```

Comment:
Looks like upstream stopped making versioned releases, thats not good.

Reply:
I decided to use this commit because it includes some string changes necessary for love2d 11 support.

Would it be preferable to downgrade love2d to version 10 and use the latest versioned release instead?

Reply:
This commit is fine. Just add the version to `releases`. I didn't notice that upstream was dead for 2 years and thought they just stopped making releases.

Reply:
I added version 1.02e to releases, it's not entirely accurate because this version is 4 commits ahead of that release.